pyproject.toml

```diff
@@ -46,7 +46,7 @@ classifiers = [
 "http" = "sceptre.template_handlers.http:Http"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<=3.12"
```

Comment:
Why extend the range to 3.12 now?

If the intention is to support Python 3.12, then I think we need to make more changes.

Did you see PR https://github.com/Sceptre/sceptre/pull/1390 ? It makes more changes to support Python 3.12.